service/controller/resource/alerting/alertmanagerconfig/test/notification-template/mimir-enabled/capi/case-1-capa-mc.golden

```diff
@@ -43,7 +42,7 @@
 {{ if (index .Alerts 0).Annotations.dashboard -}}
 📈 Dashboard: {{ template "__dashboardurl" . }}
 {{ end -}}
-👀 Prometheus: {{ template "__prometheusurl" . }}
+👀 Query: {{ template "__prometheusurl" . }}
```

Comment:
Maybe we should rename __prometheusurl to __queryurl

Reply:
Done